Documentation/kubernetes-networking.md

```diff
@@ -0,0 +1,91 @@
+# Kubernetes Networking
+
+## Network Model
+
+The Kubernetes network model outlines four methods of component communication:
+
+* Container-to-Container Communication
+    * Containers launched within the same pod can all reach each other's ports via localhost. This simplifies / secures communication between applications launched in the same Pod.
+
+* Pod-to-Pod Communication
+    * Each Pod in a Kubernetes cluster is assigned an IP in a flat shared networking namespace. This allows for a clean network model where Pods, from a networking perspective, can be treated much like VMs or physical hosts.
```

Comment:
This was just meant to explain the model (not necessarily implementations). We do discuss later, mostly by linking to the upstream "how to achieve this" page: http://kubernetes.io/v1.0/docs/admin/networking.html#how-to-achieve-this